operations/security_and_authentication/Troubleshoot_Kyverno_Configuration_manually.md

```diff
@@ -0,0 +1,25 @@
+# How to troubleshoot Kyverno configuration manually
+
+## Check if Kyverno pods are up and running
+
+Run the script with -p option.
+
+```bash
+./k8s_kyverno_pods_running.sh -p
+```
+
+This will list the expected Kyverno pods in running state.
+
+## Check if Kyverno policy report doesn’t have Failures, Warnings, Errors and Skipped policies count
+
+Run the script with -p option.
+
+```bash
+./k8s_kyverno_polr_list.sh -p
+```
+
+This is used to check the Kyverno policy report for any Failures, Warnings, Errors and Skipped policies count.
```

Comment:
Is there any guidance to give the user in the event that they do have a "failure" case on the test?  When we move it to a goss test, we will need to include something in the error message to tell the user where to get more info so that their issue can be resolved and/or how to further troubleshoot it.  Is the kyverno link below all that is needed to help with troubleshooting a non-passing policy report?